solutions/usaco-622.mdx

```diff
@@ -67,7 +67,7 @@ int main() {
 			}
 		}
 		// Update the best answer using the current dp answer
-		best_answer = min(best_answer, dp[k][0]);
+		min_dist = min(min_dist, dp[k][0]);
```

Comment:
u sure `min_dist` accurately represents the answer?

Reply:
the answer is the minimum amount of distance that all cows need to travel to fill the rooms, so I think `min_dist` is alright.